src/LitEmoji.php

```diff
@@ -183,6 +184,15 @@ public static function config(string $property, $value): void
                 self::$entityCodepoints = [];
                 self::$excludedShortcodes = [];
                 break;
+            case 'includeShortcodes':
```

Comment:
Call the configuration option `aliasShortcodes`.

Reply:
Done and documentation is up-to-date too